Azure/sonic-buildimage
# [Submodule] sonic-platform-common submodule update

#### Why I did it
xcvrd crashes in DellEMC Z9332f platform while fetching CMIS passive cable details.

        **Traceback (most recent call last):**
        File "<stdin>", line 1, in <module>
        File "/usr/local/lib/python3.9/dist-packages/sonic_platform_base/sonic_xcvr/sfp_optoe_base.py", line 24, in get_transceiver_info
        return api.get_transceiver_info() if api is not None else None
        File "/usr/local/lib/python3.9/dist-packages/sonic_platform_base/sonic_xcvr/api/public/cmis.py", line 121, in get_transceiver_info
        admin_info = self.xcvr_eeprom.read(consts.ADMIN_INFO_FIELD)
        File "/usr/local/lib/python3.9/dist-packages/sonic_platform_base/sonic_xcvr/xcvr_eeprom.py", line 30, in read
        return field.decode(raw_data, **decoded_deps)
        File "/usr/local/lib/python3.9/dist-packages/sonic_platform_base/sonic_xcvr/fields/xcvr_field.py", line 252, in decode
        result[field.name] = field.decode(raw_data[offset - start: offset + field.get_size() - start],
        File "/usr/local/lib/python3.9/dist-packages/sonic_platform_base/sonic_xcvr/fields/xcvr_field.py", line 252, in decode
        result[field.name] = field.decode(raw_data[offset - start: offset + field.get_size() - start],
        File "/usr/local/lib/python3.9/dist-packages/sonic_platform_base/sonic_xcvr/fields/xcvr_field.py", line 203, in decode
        code = struct.unpack(self.format, raw_data)[0]
        struct.error: unpack requires a buffer of 1 bytes

Issue is not seen in master branch. 
Some of the fixes from new SFP library has to be ported to 202111 branch.
   
#### How I did it
Updated sonic-platform-common sub-module to fix it. 
#### How to verify it
Check SFP related output and verify whether all the process inside PMON is running or not.
 
#### Which release branch to backport (provide reason below if selected)


#### Description for the changelog
        0f6cccd (origin/202111) [sonic_ssd] Nokia-7215: "show platform ssdhealth" not showing health percent (#279)
        b70e759 (HEAD) support new reboot-cause (#277)
        2135269 [SSD]Enhance ssd_generic with more error handling to avoid python crash #271
        393fbee [SSD Generic] Add support for parsing nvme ssd model, health and temperature (#265)
        01512ec [SSD]Enhance ssd_generic with more error handling to avoid python crash #271
        ac3e7f1 [y_cable][Broadcom] update the BRCM y_cable driver to release 2.0 (#263)
        573717a [Credo][Ycable] Fix Credo firmware download API download_firmware flag (#269)
        a844f18 [xcvr] Add get_module_fw_info method to XcvrApi class. (#267)
        35bad16 [sfputil]Refactoring read_porttab_mappings (#264)
        83c4345 [SSD Generic] Add support for parsing nvme ssd model, health and temperature (#265)
        5da31e1 [ycable][credo] Fix the is_link_active API for Credo Ycable (#260)
        931c6ea [Y-Cable][Credo] add theading locker to support thread-safe calling, add SKU check for download_firmware API.  (#222)
        8a48418 Fix SFF8472 Enhanced Options (#259)
        14006ea [ci] Fix python dependencies reference path (#262)
        937cf1a [ssd] Allow individual vendor parsers to handle errors (#252)
        ff3aa75 Fix SFF8472 Enhanced Options (#259)
        a8a83e9 [ssd] Allow individual vendor parsers to handle errors (#252)
        576f835 [component] Fix typo for FW_AUTO_ERR_UNKNOWN #254
        5985388 Add recycle port to the physical ignore list (#251)
        ee12215 split i2c read across pages (#256)
        92203d2 [component] Fix typo for FW_AUTO_ERR_UNKNOWN #254
        4b5aaf8 Add recycle port to the physical ignore list (#251)
        8e06e7f [build] Update python-wheel target path to latest in master (#253)


#### A picture of a cute animal (not mandatory but encouraged)
